From 34471140993750d4af0da3f8feef77801276db36 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 17 Sep 2021 22:36:55 +0200
Subject: [PATCH] meson: Link basuctl with libbasu dynamically

This reduces the installed size significantly. I don't know about any
reason for bundling libbasu in basuctl when they both come from the
same package.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Patch-Source: https://git.sr.ht/~emersion/basu/commit/34471140993750d4af0da3f8feef77801276db36

diff --git a/meson.build b/meson.build
index 121c0b9..056c7c4 100644
--- a/meson.build
+++ b/meson.build
@@ -330,7 +330,7 @@ exe = executable('basuctl',
                  'src/busctl/busctl-introspect.h',
                  include_directories : includes,
                  link_with : [libbasic,
-                              libsystemd_static],
+                              libsystemd],
                  install : true)
 
 meson.override_dependency('basu', declare_dependency(
-- 
2.32.0

